tests/ci/data/profiles/rosa-classic.yaml

```diff
@@ -214,4 +214,31 @@ profiles:
     oidc_config: ""
     shared_vpc: false
     imdsv2: "optional"
-    admin_enabled: false
\ No newline at end of file
+    admin_enabled: false
+- as: rosa-non-sts-local-creds
```

Comment:
Is that meant to be launched from a job ?
If yes, I don't think we need a new profile. You could have here this profile property be configurable also via an environment variable `USE_LOCAL_CREDENTIALS` which would set the profile property accordingly and configure your job to use the `rosa-non-sts-advanced` profile with the env var `USE_LOCAL_CREDENTIALS: true`

Reply:
I added a env var option now. I set it up currently so that when set to true it will enable the profile option, but if it's set to anything else it'll leave the profile option alone

Reply:
so you don't need to keep that profile here. And you would need to create a new job in `openshift/release` for this specific case